requirements.txt

```diff
@@ -0,0 +1,2 @@
+Django==1.8.17
```

Comment:
I made the requirements file according to the dependencies in the readme. It states it needs 1.8.17 for Django. Will it be fine to update the requirement in the OSM to also have 1.8.17?

Reply:
After discussion, we should stick with 1.8.14. Adjustments will be made for the PR.